### DISCUSSION TITLE
Audio Quality

### DISCUSSION THREAD

**Discussion by abbysheksaini** [Shabinder/SpotiFlyer#50] [General]
In which quality does it download from Spotify 128 or 256 or 320 kbps?

**Comment by Shabinder** [comment]
It downloads from youtube using spotify details and metadata ,where mostly songs are of around **160 kbps.**

**Reply by maurix99** [reply]
This cant be better? 
thanks, good app!

**Reply by PatrikKnowhow** [reply]
If you could do something like TuneFab Spotify Music Converter for Android, it would be awesome. Ripping from already compressed Youtube isn't that great. Thank you anyway!

**Reply by ShapeShifter499** [reply]
@Shabinder Ah so this isn't using some trick to download from spotify directly

**Comment by Ayushp31** [comment]
The audio quality should be 320 kbps MP3 or selective options where we have to choose the audio quality

**Reply by Shabinder** [reply]
Yes it should be like that , but as of now ,youtube is our only source which sadly doesn't provide 320kbps.

**Reply by rajeshrvds** [reply]
Can u try 320kbps in Spotify?

**Comment by Ayushp31** [comment]
You can try with Spotify or other apps

**Comment by ghost** [comment]
Support for m4a audio file

**Reply by Shabinder** [reply]
I can do that , but there is no reliable library for metadata tags editing.
Jaudio tagger and so have issues with youtube m4a files.

**Comment by ghost** [comment]
Hello and thank you for your hard work!
My question is, 
Premium spotify 320kbps quality 
Free spotify 128!

My question is if I'm free spotify does spotiflyer reflect the change in quality

Or am I misunderstanding? I AM LEARNING...
